packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -135,11 +135,7 @@ export class DhisService extends Service {
     const { programCodes } = options;
 
     if (programCodes) {
-      // TODO remove `useDeprecatedApi` option as soon as `pullAnalyticsFromEventsForApi_Deprecated()` is deleted
-      const { useDeprecatedApi = true } = options;
-      return useDeprecatedApi
-        ? this.pullAnalyticsFromEventsForApi_Deprecated
-        : this.pullAnalyticsFromEventsForApi;
+      return this.pullAnalyticsFromEventsForApi;
```

Comment:
This commit would be a great separate PR. It would also involve regression testing a whole bunch of visualisations right?